src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -0,0 +1,767 @@
+	"help.securitycraft.ims.info": "Le S.M.I, ou Système de Munitions Intelligent, est une mine à quatre utilisations qui ciblera les joueurs (ou mobs) qui entrent dans son rayon de tir. Une fois qu’une mine est lancée, elle traquera l’entité et explosera. Les mines peuvent être rechargées en cliquant avec le bouton droit sur S.M.I. avec une mine rebondissante. Une fois les quatre mines utilisées, le S.M.I. va s’autodétruire.",
+	"help.securitycraft.protecto.info": "Le protecto, inspiré par l’ennemi du jeu \"Megaman: Battle Network 2 \" de Capcom, attaquera tout mob dans un rayon de 10 blocs avec des éclairs toutes les 10 secondes. Il doit pleuvoir et le ciel doit être visible pour que ça marche.",
+	"help.securitycraft.motion_activated_light.info": "La lumière à détecteur de mouvement émettra une lumière, semblable à une lampe redstone, lorsqu’un joueur se trouve à une certaine portée du bloc (configurable).",
+	"help.securitycraft.trophy_system.info": "inspiré par le gadget vu pour la première fois dans \"Call of Duty: Modern Warfare 3 \" publié par Activision, cible les flèches et les boules de feu ennemies et les détruira dans la portée après un temps de charge d’une demi-seconde",
+	"help.securitycraft.crystal_quartz.info": "Les blocs de Cristal de Quartz sont de simple blocs de construction qui peuvent être blindés de la façon habituelle.",
+	"help.securitycraft.reinforced_crystal_quartz.info": "Les bloc de cristal de quartz blindés sont utilisés pour construire le bloc de poche. Plus d’informations sur la page du gestionnaire du bloc de poche.",
+	"help.securitycraft.block_pocket_wall.info": "Les murs de Bloc de poche sont utilisés pour construire le bloc de poche. Plus d’infos sur la page du Gestionnaire de Bloc de poche.",
+	"help.securitycraft.block_pocket_manager.info": "Le Gestionnaire de Bloc de poche est utilisé pour gérer le Bloc de poche, qui est une pièce dans laquelle seul son propriétaire et les joueurs sur liste blanche peuvent entrer. Lorsque vous ouvrez son interface graphique, vous voyez une sélection de tailles. Choisissez-en une, puis construisez le Bloc de poche autour du gestionnaire. Assurez-vous que le gestionnaire est placé à l’extérieur de la couche inférieure. Le Bloc de poche est un cube de la taille que vous avez choisie. Les coins sont construits avec du cristal de Quartz sculpté blindé, les blocs entre les coins sont construits avec des piliers de cristal de Quartz blindé correctement orientés, et les sol / mur / plafond sont construits avec des murs de Bloc de poche. Voir cette image pour un exemple de son apparence: https://i.imgur.com/opc7dqO.png - Cliquer sur \"Activer \" après la construction de la structure activera tous les modules ajoutés. Voir l’interface graphique de personnalisation pour plus d’informations (clic droit avec ModificateUr de Bloc universel).",
```

Comment:
Remove the space in front of the second `\`.

Reply:
There's also a capital `U` at the end in `ModificateUr` which I'm assuming should not be there.